src/Uno.UI/UI/Xaml/UIElement.Keyboard.skia.cs

```diff
@@ -67,13 +82,6 @@ private void CoreWindow_KeyUp(CoreWindow sender, KeyEventArgs args)
 						$"ScanCode: {args.KeyStatus.ScanCode})"
 					);
 				}
-
-				var originalSource = FocusManager.GetFocusedElement() as UIElement ?? Window.Current.Content;
```

Comment:
The containing method only does logging and not anything useful. Should it be deleted, and possibly the logging moved to another relevant place (e.g., your new methods)?

Reply:
Well, I thought I would leave it in case more logic needs to be added later. Theoretically, a CoreWindow might want to run its own logic on KeyUp/Down. Currently no logic actually exists (except logging), so we can do it either way. This would also mean CoreWindow.KeyUp/Down will be empty, which isn't pretty.